tests/Doctrine/Tests/DbalFunctionalTestCase.php

```diff
@@ -40,7 +40,7 @@ protected function setUp()
         $this->_conn->getConfiguration()->setSQLLogger($this->_sqlLoggerStack);
     }
 
-    protected function onNotSuccessfulTest($e)
+    protected function onNotSuccessfulTest(\Exception $e)
```

Comment:
This is weird, I am not getting this notice and [neither does PHPUnit have a type hint there](https://github.com/sebastianbergmann/phpunit/blob/5.4.6/src/Framework/TestCase.php#L2237)...